app/move/views/confirmation.njk

```diff
@@ -30,8 +30,8 @@
         {% set context = "assign" %}
       {% else %}
 
-        {% if move.status === "proposed" and (move.move_type === "secure_training_centre" or move.move_type === "secure_childrens_home") %}
-          {% set context = move.move_type %}
+        {% if move.status === "proposed" and (move.from_location.location_type === "secure_training_centre" or move.from_location.location_type === "secure_childrens_home") %}
```

Comment:
There's a lot of these `move.move_type` to `move.from_location.location_type` changes - what the difference between this and the `move?.to_location?.location_type` on the SCH removal above?

Reply:
Yes, I've tried to explain this within the [commit message](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/1001/commits/a7efddb5e67a08870ebbcfd7fc258d4a9200f366) as to why these changes are happening.

Essentially, I've removed the value from the API being updated in two places and used a different piece of date (`from_location.location_type`) to provide the same override setup to the translation library.

Reply:
Commit message links don't survive rebasing off master :wink: 

Anyway, that makes sense (as does the commit messages)